### PR TITLE
[vtadmin] RBAC Documentation

### PR DESCRIPTION
**Pull request by ajm188**

### CODE REVIEW

File: content/en/docs/14.0/reference/vtadmin/role-based-access-control.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+	// Returning an error from the authenticator will fail the request. To
+	// denote an authenticated request, return (nil, nil) instead.
+	AuthenticateHTTP(r *http.Request) (*Actor, error)
+}
+```
+<br/>
+
+If running with an authentication plugin installed, VTAdmin will invoke its `Authenticate` method on all incoming gRPC requests, and its `AuthenticateHTTP` method on all incoming HTTP requests.
+
+Returning an error from either of these methods will fail the request with either an `UNAUTHENTICATED` code for gRPC requests and an `UNAUTHORIZED` response for HTTP requests.
```
Comment (deepthi) [inline content/en/docs/14.0/reference/vtadmin/role-based-access-control.md]: The second "either" isn't necessary

File: content/en/docs/14.0/reference/vtadmin/role-based-access-control.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+  - resource: "*"
+    actions:
+    - "get"
+    - "ping"
+    subjects: ["*"]
+    clusters: ["*"]
+
+  - resource: "*"
+    actions:
+	- "create"
```
Comment (deepthi) [inline content/en/docs/14.0/reference/vtadmin/role-based-access-control.md]: The indentation on the yaml example looks off. Can you check and fix that?

**Review by deepthi — APPROVED** [review]
This is a superbly written document. Once the two minor points are addressed for both versions you may go ahead and merge it.

File: content/en/docs/14.0/reference/vtadmin/role-based-access-control.md (hunk 2 above)
Comment (ajm188) [reply content/en/docs/14.0/reference/vtadmin/role-based-access-control.md]: yep! gonna blame some combination of vscode and "yaml inside a .md file" for this one

File: content/en/docs/14.0/reference/vtadmin/role-based-access-control.md (hunk 1 above)
Comment (ajm188) [reply content/en/docs/14.0/reference/vtadmin/role-based-access-control.md]: good catch!

**Pull request merged**